src/module.c

```diff
@@ -5623,9 +5623,17 @@ RedisModuleString *RM_CreateStringFromCallReply(RedisModuleCallReply *reply) {
     }
 }
 
-/* Object to correspond to NULL/Root User, to distinguish from NULL (unset/reset) value */
+/* Object to correspond to Redis' Unrestricted User, to distinguish from NULL (unset/reset) value */
 const RedisModuleUser NullUser = {0};
 
+/* By default, RM_Call executes commands as unrestricted user.  Even, when one uses the "C"
+ * ACL check flag,  it only enforces the check on the provided command itself.  ACL checks
+ * performed by the command itself during execution, such as with scripts, would by default,
+ * still run as the unrestricted user.
+ *
+ * This API changes which user the ACL check is performed against, as well as causes Redis
+ * toexecute the command as the same user thereby enabling ACL checks performed by the command
```

Comment:
"toexecute".
p.s. i don't think we're using double spaces after periods elsewhere.

Reply:
took out the the comment here and replaced it in a much simpler manner.

/* Modifies the user that RM_Call will ACL check commands as */